monsteer/steering/simulatorPlugin.h

```diff
@@ -25,19 +25,16 @@ namespace monsteer
 {
 
 /** Init data for SimulatorPlugin
- * version 0.2
+ * @version 0.4
  */
 class SimulatorPluginInitData
 {
 public:
-    explicit SimulatorPluginInitData( const URI& subscriber_,
-                                      const URI& publisher_ )
+    explicit SimulatorPluginInitData( const URI& subscriber_ )
         : subscriber( subscriber_ )
-        , publisher( publisher_ )
     {}
 
     const URI subscriber;
-    const URI publisher;
```

Comment:
It could be more useful here. But I am not sure if we are really going to use the session parameter programmatically in practice, or just use the UI to do the filtering and give a fully-qualified uri to the plugin, so I left it out for now